style.css

```diff
@@ -733,15 +743,15 @@ ul li {
   border-radius: 4px;
 }
 
-.contact-btn:enabled {
+.contact-btn:active {
```

Comment:
- [ ] I think it's better to use the properties for active provided by the Figma template. It's a project requirement.
- [ ] Same thing for "Get my resume" button.
- [ ] Same thing for "See project" button.

<img width="667" alt="Screen Shot 2022-08-26 at 10 38 41 PM" src="https://user-images.githubusercontent.com/47673311/187009266-4b8d465f-02d2-4a2a-9159-c5b22b7f5374.png">